src/App.jsx

```diff
@@ -13,6 +13,7 @@ function App() {
         <Route path="/" element={<Home />}/>
         <Route path="/home" element={<Home />}/>
         <Route path="/articles" element={<ArticleList />} />
+        <Route path="/articles/:articleid" element={<Article />} />
```

Comment:
Please check sure you are consistent with naming as here the 'articleid' is not camel-cased, but in your other component it is. This will be important when you refactor your code to use 'useParams'. See my comment below.